### PR TITLE
PR Fixes for Pocket Pages

### PR DESCRIPTION
**Pull request by reemhamz**
## Description
Working on the PR comments left here: https://github.com/mozilla/bedrock/pull/10645 to be added onto https://github.com/mozilla/bedrock/pull/10828. Unfortunately the former got botched I'm working on fixing things up in the latter.

## Issue / Bugzilla link
Original PR holding PR comments: https://github.com/mozilla/bedrock/pull/10645

### CODE REVIEW

File: bedrock/externalpages/templates/externalpages/pocket/base.html [hunk 1]
```diff
@@ -26,6 +25,9 @@
       <meta property="og:title" content="{{ self.page_title() }} | Pocket" />
       <meta property="og:description" content="{{ self.page_desc()}}"/>
       <meta property="og:site_name" content="Pocket" />
+      <link rel="icon" type="image/png" sizes="196x196" href="{% block page_favicon %}{{ static('img/favicons/pocket/favicon.ico') }}{% endblock %}">
+      <link rel="icon" type="image/png" sizes="196x196" href="{% block page_favicon_large %}{{ static('img/favicons/pocket/favicon-196x196.png') }}{% endblock %}">
+       <link rel="shortcut icon" href="{% block page_ios_icon %}{{ static('img/favicons/mozilla/apple-touch-icon.png') }}{% endblock %}">
```
Comment (alexgibson) [inline bedrock/externalpages/templates/externalpages/pocket/base.html:30]: I think can be simplified to:

```jinja
<link rel="icon" type="image/png" sizes="196x196" href="{% block page_favicon_large %}{{ static('img/favicons/pocket/favicon-196x196.png') }}{% endblock %}">
<link rel="shortcut icon" href="{% block page_favicon %}{{ static('img/favicons/pocket/favicon.ico') }}{% endblock %}">
```

File: media/css/externalpages/pocket/components/_platform-footer.scss [hunk 2]
```diff
@@ -36,8 +36,13 @@ $image-path: '/media/protocol/img';
                 margin: 0 0 $spacing-md;
                 cursor: pointer;
 
-                a:hover {
-                    text-decoration: underline;
+                a {
+                    color: $color-text-primary;
+                    text-decoration: none;
+
+                    &:hover {
```
Comment (alexgibson) [inline media/css/externalpages/pocket/components/_platform-footer.scss:43]: Nit: include `active` and `focus` here too?

File: requirements/dev.txt [hunk 3]
```diff
@@ -28,26 +28,32 @@ pyquery==1.4.0 \
...
-    --hash=sha256:a1b73c7c4d2a42b9d37dd43199c5711d91424ff3c6c22681bc132db4a4afec6f \
-    --hash=sha256:abbff240f77347d17306d3201e14431519bf64495648ca5a49571f988f88dee9 \
-    --hash=sha256:ad9b8c1206ae41d46ec7380b78ba735ebb77758a650643e841dd3894966c31d0 \
-    --hash=sha256:c9cd3828bbe1a40070c11fe16a51df733fd2f0cb0d745fb83b7b5c1f05967df7 \
-    --hash=sha256:cd1cf1deb3d5544bd942356364a2fdc8959bad2b6cf6eb17f47d301ea34ae822 \
-    --hash=sha256:d036dc1ed8e1388e995833c62325df3f996675779541f682677efc6af71e96cc \
-    --hash=sha256:e4fb7ced4d9dec77d6cf533acfbf8e1415fe799430366affb18d69ee8a3c6330 \
-    --hash=sha256:e7a0b42db2a47ecb488cde14e0f6c7679a2c5a9f44814393b162ff6397fcdfbb \
-    --hash=sha256:f2f184bf38e74f152eed7f87e345b51f3ab0b703842f447c22efe35e59942c24
+coverage==6.1.2 \
```
Comment (alexgibson) [inline requirements/dev.txt:31]: This doesn't seem related to this PR, so I'd suggest reverting this file to whatever is in the master branch here.

**Review by alexgibson — CHANGES_REQUESTED** [review]

File: media/css/externalpages/pocket/components/_footer.scss [hunk 4]
```diff
@@ -4,11 +4,14 @@
 
 @import '../utils/variables';
 
-$image-path: '/media/img/externalpages/pocket';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+$image-path: '/media/protocol/img';
+$pocket-image-path: '/media/img/externalpages/pocket';
```
Comment (reemhamz) [inline media/css/externalpages/pocket/components/_footer.scss:10]: I had to use two different image paths, so I named the one holding pocket paths $pocket-image-path. However, in other files, we do have $image-path holding the pocket paths.

**Review by alexgibson — APPROVED** [review]

**Pull request merged**